docs/tooling/recording_and_replay.md

```diff
@@ -8,11 +8,11 @@ A started communication server during replay can be used to investigate the reco
 ## Record
 
 - Manual upload to a robot
-    - Use `./pepsi recording ...` to enable recording for certain cycler instances, e.g., `./pepsi recording Control VisionTop`
-        - This will set the cycler instances in `etc/parameters/framework.json`
+    - Use `./pepsi recording ...` to enable recording at different recording rates, e.g., `./pepsi recording Control=1,VisionTop=30`
+        - This will set the cycler instances and recording rate in `etc/parameters/framework.json`
     - Use `./pepsi upload ...` to upload as usual (this includes the framework configuration with the cycler instances)
 - Pregame
-    - Use `./pepsi pregame --cycler-instances-to-be-recorded ... ...` to enable recording for certain cyclers and upload to the robot in one step, e.g., `./pepsi pregame --cycler-instances-to-be-recorded Control,VisionTop ...`
+    - Use `./pepsi pregame --recording-settings ... ...` to enable recording at different recording rates and upload to the robot in one step, e.g., `./pepsi pregame --recording-settings Control=1,VisionTop=30 ...`
```

Comment:
Also rename the command line arguments in the documentation?

Reply:
Good catch, fixed.